frontend/src/components/GddSend/TransactionConfirmationSend.spec.js

```diff
@@ -45,6 +45,33 @@ describe('GddSend confirm', () => {
       it('renders the component div.confirm-box-send', () => {
         expect(wrapper.find('div.confirm-box-send').exists()).toBeTruthy()
       })
+
+      describe('send now button', () => {
+        beforeEach(() => {
+          jest.clearAllMocks()
+        })
+
+        describe('single click', () => {
+          beforeEach(async () => {
+            await wrapper.find('button.btn-success').trigger('click')
+          })
+
+          it('emits send transaction one time', () => {
+            expect(wrapper.emitted('send-transaction')).toHaveLength(1)
+          })
+        })
+
+        describe('double click', () => {
+          beforeEach(async () => {
+            await wrapper.find('button.btn-success').trigger('click')
+            await wrapper.find('button.btn-success').trigger('click')
```

Comment:
I did this test on master without the fix of @ogerly and it fails, as the event is emitted twice. I approve the solution.